test/warnings-test.js

```diff
@@ -206,3 +207,16 @@ testHelpers.dev.devOnlyTest("Variable not found warning should suggest correct k
 	Scope.prototype.getPathsForKey = origGetPaths;
 
 });
+
+testHelpers.dev.devOnlyTest("Should give a warning when a partial is not found #493", function () {
+	var template,
+		items = new DefineList(['one', 'two']),
+		teardown = testHelpers.dev.willWarn('Unable to find partial "itemRenderer"');
+
+	template = stache("{{#each(items)}} {{> itemRenderer}} {{/each}}");
```

Comment:
Why is each needed?  Also `DefineList`?  If there's a way to do this with more minimal example, please do so.

Reply:
Done!